pkg/instancegroups/rollingupdate_os_test.go

```diff
@@ -60,7 +60,7 @@ func getTestSetupOS(t *testing.T, ctx context.Context) (*RollingUpdateCluster, *
 
 	assetBuilder := assets.NewAssetBuilder(inCluster.Spec.Assets, inCluster.Spec.KubernetesVersion, false)
 	basePath, _ := vfs.Context.BuildVfsPath(inCluster.Spec.ConfigBase)
-	clientset := vfsclientset.NewVFSClientset(basePath)
+	clientset := vfsclientset.NewVFSClientset(vfs.Context, basePath)
```

Comment:
We may need to think of a better name for this vfsclientset.  Maybe `StateStoreClient` (the alternative is when we back with kubernetes CRDs, which is still experimental).  Not a blocker though!

Reply:
We could call it `VFSSimpleClientset` or `VFSKopsClientset`